src/status_im/ui/screens/profile/events.cljs

```diff
@@ -52,7 +52,10 @@
  :my-profile/update-picture
  (fn [{:keys [db]} [this-event base64-image]]
    (if base64-image
-     {:db (assoc-in db [:my-profile/profile :photo-path] (str "data:image/jpeg;base64," base64-image))}
+     {:db       (-> db
+                    (assoc-in [:my-profile/profile :photo-path]
+                              (str "data:image/jpeg;base64," base64-image))
+                    (assoc :my-profile/editing? true))}
```

Comment:
I take this is fixing some existing issue, maybe add that to the PR description as it's not clear if related to this